src/main/java/com/aws/greengrass/mqtt/bridge/clients/LocalMqtt5Client.java

```diff
@@ -419,7 +451,7 @@ void subscribe(String topic) throws RetryableMqttOperationException {
         LOGGER.atDebug().kv(LOG_KEY_TOPIC, topic).log("Subscribing to MQTT topic");
         try {
             SubAckPacket subAckPacket = client.subscribe(subscribePacket)
-                    .get(MQTT_OPERATION_TIMEOUT_MS, TimeUnit.MILLISECONDS);
+                    .get(ackTimeoutSeconds, TimeUnit.SECONDS);
```

Comment:
do we need the get here given that we are setting the ack timeout in the client itself now?

Reply:
https://github.com/awslabs/aws-crt-java/blob/7cce500824acd81cda9cb8e0453fc9a08b832f38/src/main/java/software/amazon/awssdk/crt/mqtt5/Mqtt5ClientOptions.java#L788-L789

Reply:
still need a get because LocalMqtt5Client isn't async yet, but yeah we'll want to remove the gets eventually